### PR TITLE
Nachooo!

### PR DESCRIPTION
**Pull request by FredThesing**
- Add Nacho emoticon.

### CODE REVIEW

**Comment by adrianpike** [comment]
natcho buddy guy

:+1:

**Comment by gaassa** [comment]
:+1:  NAAAAAAAACHHHHOOOOOO!!!! :BI (eagle powers)

**Comment by codebender** [comment]
:-1: Max dimensions are 120px tall and 120px wide, can't be less than 64px

**Comment by rubysolo** [comment]
get that corn(y icon) outta my face!
:+1: (after fixing dimensions)

File: README.md [hunk 1]
```diff
@@ -79,6 +79,7 @@ We are using HipChat's built-in emoticon feature, which means an admin must uplo
 | ![mustache](emoticons/mustache.png)               | mustache        |
 | ![mustache2](emoticons/mustache2.png)             | mustache2       |
 | ![mustache3](emoticons/mustache3.png)             | mustache3       |
+| ![nacho](emoticons/nacho.png)                     | nacho           |
```
Comment (codebender) [inline README.md]: .png?

**Pull request merged**